src/Experimental/Traversals/bfs.jl

```diff
@@ -78,18 +79,20 @@ starting at vertex `s` (vertices `ss`).
 """
 function visited_vertices(
     g::AbstractGraph{U},
-    ss::AbstractVector{U},
+    ss::AbstractVector,
```

Comment:
would you want this to be `AbstractVector{<: Integer}` instead?

Reply:
I just broadened it to `AbstractVector` - let it error if it doesn't get `<:Integer`.